src/Worker.Extensions.DurableTask/Worker.Extensions.DurableTask.csproj

```diff
@@ -29,7 +29,7 @@
     <AssemblyOriginatorKeyFile>..\..\sign.snk</AssemblyOriginatorKeyFile>
 
     <!-- Version information -->
-    <VersionPrefix>0.4.1</VersionPrefix>
+    <VersionPrefix>0.5.0</VersionPrefix>
     <VersionSuffix>beta</VersionSuffix>
```

Comment:
Should we consider elevating this to "rc" status (instead of "beta") since we think this is pretty close to what the final release will look like? I don't have a strong opinion either way. We'd probably need to do this for both the core SDK and this extension version...